portfolio-item.html

```diff
@@ -0,0 +1,81 @@
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Dev David Portfolio</title>
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/7.0.0/normalize.min.css">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+        <!-- <script src="https://kit.fontawesome.com/66164385f4.js" crossorigin="anonymous"></script> -->
+        
+        <!-- Fonts -->
+        <link href="https://fonts.googleapis.com/css?family=Source+Code+Pro:400,900|Source+Sans+Pro:300,900&display=swap" rel="stylesheet"> 
+        <link rel="stylesheet" href="css/index.css"
+    </head>
+    <body>
+        <header>
+            <div class="logo">
+                <img src="img/devjane.png" alt="">
+            </div>
+            <button class="nav-toggle" aria-label="toggle navigation">
+                <span class="hamburguer"></span>
+            </button>
+            <nav class="nav">
+                <ul class="nav__list">
+                    <li class="nav_item"><a href="index.html" class="nav__link">Home</a></li>
+                    <li class="nav_item"><a href="index.html#services" class="nav__link">My Services</a></li>
+                    <li class="nav_item"><a href="index.html#about" class="nav__link">About me</a></li>
+                    <li class="nav_item"><a href="index.html#work" class="nav__link">My Work</a></li>
+                </ul>
+            </nav>
+        </header>
+        
+        <!-- INTRODUCTION -->
+        <section class="intro" id="home">
+             <h1 class="section__title section__title-intro">
+                 The title <strong>of my project</strong>
+            </h1>
+             <p class="section__subtitle section__subtitle--intro">A short subtitle</p> 
+             <img src="img/portfolio-01.jpg" alt="a picture of David" class="intro__img">    
+        </section>
+        
+        <!-- PORTFOLIO ITEM -->
+        
+        <div class="portfolio-item-individual">
+            <p >Voluptatibus, soluta blanditiis! Incidunt ea unde itaque illo molestiae eligendi sint culpa nobis voluptas sapiente voluptate, magnam ipsum eius earum?</p>
+            <img src="img/project-details.jpg" alt="">
+            <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Saepe et amet tenetur! Fugit sequi optio corrupti fugiat ducimus consequatur incidunt?</p>
+            <p>Voluptatibus, soluta blanditiis! Incidunt ea unde itaque illo molestiae eligendi sint culpa nobis voluptas sapiente voluptate, magnam ipsum eius earum?</p>
+        </div>
+        
+        
+        <!-- FOOOTER -->
+        
+        <footer class="footer">
+            <a href="mailto:david.briseo@gmail.com" class="footer__link">david.briseo@gmail.com</a>
```

Comment:
Learn about environment variables and how you can integrate them into your code. I think it's a good practice to not hardcode emails at .html or .js files. so find a way to integrate them into your code.